.changeset/fuzzy-crews-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+`GovernorSuperQuorum`: Add a governance extension to support a super quorum. Proposals that meet the super quorum can be executed earlier than the proposal deadline.
```

Comment:
```suggestion
`GovernorSuperQuorum`: Add a governance extension to support a super quorum. Proposals that meet the super quorum can be queued earlier than the proposal deadline.
```
Not necessarily accurate as is given that the timelock delay still applies.

Reply:
```suggestion
`GovernorSuperQuorum`: Add a governance extension to support a super quorum. Proposals that meet the super quorum advance to the `Succeeded` state before the proposal deadline.
```

What about this?

Reply:
actually, none of the proposal are correct. When the super quorum is met, it advances to the succedded state only if the voting has a majority of for.

Reply:
```suggestion
`GovernorSuperQuorum`: Add a governance extension to support a super quorum. Proposals that meet the super quorum (and have a majority of for votes) advance to the `Succeeded` state before the proposal deadline.
```

Reply:
Ok good. Let's change all short descriptions of the contract to this.